pkg/gits/github.go

```diff
@@ -343,6 +343,8 @@ func (p *GitHubProvider) CreateRepository(owner string, name string, private boo
 		}
 		return nil, errors.New(msg)
 	}
+	// Sleep 5 seconds to ensure repository exists enough to be pushed to.
+	time.Sleep(5 * time.Second)
```

Comment:
hmmm, for what it's worth, imo sleeping is just a crutch. I'd try to do an active poll loop in some form.